packages/client/src/components/form/text-area/TextArea.tsx

```diff
@@ -0,0 +1,20 @@
+import { TextareaHTMLAttributes, forwardRef } from 'react';
+
+import { StyledFormControl, StyledFormLabel } from '../input/Input.styled';
+
+import { StyledFormTextArea } from './TextArea.styled';
+
+const TextArea = forwardRef<HTMLTextAreaElement, TextareaHTMLAttributes<HTMLTextAreaElement>>((props, ref) => {
+  return (
+    <>
+      <StyledFormControl>
+        <StyledFormLabel>Bio</StyledFormLabel>
```

Comment:
`Bio` should be label prop(dynamic) & if you forward the label value, then & only then, you should render the label.